src/opera/commands/deploy.py

```diff
@@ -21,14 +21,24 @@ def add_parser(subparsers):
     )
     parser.add_argument(
         "--inputs", "-i", type=argparse.FileType("r"),
-        help="Optional: YAML file with inputs to override the inputs supplied in init",
+        help="Optional: YAML file with inputs to "
+             "override the inputs supplied in init",
     )
     parser.add_argument(
         "--workers", "-w",
-        help="Maximum number of concurrent deployment \
-            threads (positive number, default 1)",
+        help="Maximum number of concurrent deployment "
+             "threads (positive number, default 1)",
         type=int, default=1
     )
+    parser.add_argument(
+        "--resume", "-r", action='store_true',
+        help="Resume the deployment from where it was interrupted.",
+    )
+    parser.add_argument(
+        "--force", "-f", action='store_true',
+        help="Delete the previous deployment state "
+             "and start over the deployment.",
```

Comment:
Force is a bit awkward for me. As soon as I saw the prompts below, I thought --force just skipped them. But no, it deletes the previous deployment. How about `-d/--delete-existing`?

Reply:
Yeah, maybe force can be confusing from the perspective of those who look into the code, but for opera CLI users probably not since they would force the deployment. What do you think @matejart, @cankarm, @dradx?

Reply:
But I'd expect "forcing" to just do what it did previously, before these checks were introduced. Maybe not though, I'm really not a user of xopera :)

Reply:
Before this PR we had no forcing and if you tried to deploy again on already deployed instance nothing really happened. But `-d/--delete-existing` could be if everyone thinks it's ok.

Reply:
@sstanovnik The root of this issue lies in statefulness of xOpera. The orchestrator saves the states and if you deployed something, then you do a manual change on deployed infra or maybe a small change of some module config, another xopera deploy would ignore that. Deploy would check the state written in .opera and skip all successful steps.

And then there is an argue what is _normal expected_ behaviour for `deploy`. Should always start to redeploy something regardless of the state in .opera or skip successful steps and just resume... this long discussion is in issue #46 . We decided to use switches that exactly explain what happens.

So `-d/--delete-existing` seems promising, but what exactly would be, delete existing. This force actually deletes/ignores only opera state (so .opera) and not "delete deployed instances". This only means that ansible scripts will be re-run on all instances.

Reply:
I would agree with this implementation by introducing `--force/-f `  and `--resume/-r` - these are nice features, but as I pointed out in a previous comment I really don't see any point of prompting users after he/she expresses a clear request by using these switches. 

Otherwise I agree with @sstanovnik that this features should be made available in API (library) calls. The `--resume/-r` seems especially interesting in HPC long running jobs scenarios - when something goes wrong and the user wishes not to repeat the successfully executed jobs.